self-tests.yml

```diff
@@ -576,6 +598,7 @@ tasks:
     depends_on:
       - name: dist
         status: success
+    disable: true
```

Comment:
I know this PR is WIP but reminder we can't commit this

Reply:
oops ill be sure to delete this